x/ccv/staking/doc.go

```diff
@@ -0,0 +1,12 @@
+/*
+Package staking defines a "wrapper" module around the Cosmos SDK's native
+x/staking module. In other words, it provides the exact same functionality as
+the native module in that it simply embeds the native module. However, it
+overrides two core methods, `InitGenesis` and `EndBlock`. Specifically, these
+methods perform no-ops and return no validator set updates, as validator sets
+are tracked by the consumer chain's x/ccv/consumer module.
+
+The consumer chain should utilize the x/ccv/staking module to perform democratic
+actions such as participating and voting within the chain's governance system.
+*/
+package staking
```

Comment:
should we rename this? its as confusing as tendermint the algorithm,  tendermint the software, tendermint the company

Reply:
I thought about this, but see my comment: https://github.com/cosmos/interchain-security/pull/89#discussion_r866852714